package/widevine/widevine.mk

```diff
@@ -3,7 +3,7 @@
 # widevine
 #
 ################################################################################
-WIDEVINE_VERSION = a4998dc60e358c436ef56577f94ac2ae793f09ce
+WIDEVINE_VERSION = 315655c6ebf58a696b6dde78b14db7aad527e32f
```

Comment:
What's this about? Seem unrelated to the patch

Reply:
Bumbed the widevine to the latest to get env changes